feature1.txt

```diff
@@ -0,0 +1 @@
+This feature was created in feature1 branch
```

Comment:
It's really good that you decided to write this line of code.

Reply:
Thank's for this comment.